functions/index.js

```diff
@@ -72,4 +72,39 @@ app.get("/users/:email", async (req, res) => {
   functions.logger.log(docs[0]);
   return res.status(200).json(docs[0]);
 });
+
+app.get("/users", async (req, res) => {
+  functions.logger.log("GET /users");
+
+  const snapshot = await db.collection("users").get();
+  if (snapshot.empty) {
+    functions.logger.log("No matching documents");
+    return res.status(404).send();
```

Comment:
If the error code is the same, it would be kinder for the user to align with the same error message.

{ message: "Not Found" }